Wikipedia/Code/PageHistoryFetcher.m

```diff
@@ -73,7 +73,7 @@ - (NSDictionary*)getParamsForTitle:(MWKTitle*)title {
         @"action": @"query",
         @"prop": @"revisions",
         @"rvprop": @"ids|timestamp|user|size|parsedcomment",
-        @"rvlimit": @50,
+        @"rvlimit": @51,
```

Comment:
@ndis1 can you explain why this is now 51 instead of 50 and how this fixes the issue?